portray config --sync fails if no config exists

If I run `portray config --sync` with no `.portray.yaml` I get the following error:

```
$ portray config --sync
panic: Config File ".portray" Not Found in "[/etc/portray /Users/rharrigan]"

goroutine 1 [running]:
github.com/jasonamyers/portray/cmd.check(0x15f1c20, 0xc420133220)
    /Users/rharrigan/gocode/src/github.com/jasonamyers/portray/cmd/config.go:215 +0x4a
github.com/jasonamyers/portray/cmd.glob..func2(0x1859160, 0xc4200b93b0, 0x0, 0x1)
    /Users/rharrigan/gocode/src/github.com/jasonamyers/portray/cmd/config.go:69 +0x2b
github.com/jasonamyers/portray/vendor/github.com/spf13/cobra.(*Command).execute(0x1859160, 0xc4200b9390, 0x1, 0x1, 0x1859160, 0xc4200b9390)
    /Users/rharrigan/gocode/src/github.com/jasonamyers/portray/vendor/github.com/spf13/cobra/command.go:702 +0x2c6
github.com/jasonamyers/portray/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x1859380, 0x155db67, 0x7, 0xc4201fa460)
    /Users/rharrigan/gocode/src/github.com/jasonamyers/portray/vendor/github.com/spf13/cobra/command.go:783 +0x2e4
github.com/jasonamyers/portray/vendor/github.com/spf13/cobra.(*Command).Execute(0x1859380, 0xc4200ae058, 0x0)
    /Users/rharrigan/gocode/src/github.com/jasonamyers/portray/vendor/github.com/spf13/cobra/command.go:736 +0x2b
github.com/jasonamyers/portray/cmd.Execute()
    /Users/rharrigan/gocode/src/github.com/jasonamyers/portray/cmd/root.go:45 +0x2d
main.main()
    /Users/rharrigan/gocode/src/github.com/jasonamyers/portray/main.go:25 +0x20
```

This can be worked around with `touch ~/.portray.yaml`

--- a/cmd/config.go
+++ b/cmd/config.go
@@ -65,8 +65,6 @@ var configCmd = &cobra.Command{
 	Long: `The config command allows you to view the current Portray config,
 as well as sync it with the AWS CLI config`,
 	Run: func(cmd *cobra.Command, args []string) {
-		err := viper.ReadInConfig() // Find and read the config file
-		check(err)
 
 		if viper.GetBool("sync") {
 			//fmt.Println("Attempting to parse ~/.aws/config")

